tests/test_other.py

```diff
@@ -11886,6 +11886,7 @@ def test_wasmfs_getdents(self):
 
   @wasmfs_all_backends
   def test_wasmfs_readfile(self):
+    self.set_setting('WASM_BIGINT')
```

Comment:
Please add `@parameterized` on this test, so we don't lose the non-bigint testing coverage.

Reply:
Probably `@parameterized` will not work in combination with `@wasmfs_all_backends` (I could not combine it with `@also_with_wasm_bigint` either). I just split it to a separate function with commit 567cfd3cf97835be64c482866ac21523ad6ca68d.

Reply:
How about adding an optional `wasm_bigint` parameter to the `wasmfs_all_backends` decorator instead? See e.g. commit f19dcaddbe437b53752199d22c11eed368cd1f1f.

Reply:
I'm slightly concerned about composability, if we end up passing more and more flags to `wasmfs_all_backends` in the future. But to be fair we might not have any more that we need. So I'd be ok with this for now, and we can refactor later if we need more.

Reply:
Good point. An alternative would be to move this test to `test_core.py`, which includes a `bigint.*` test suite.
https://github.com/emscripten-core/emscripten/blob/58265624de240e238a75604e461f54f4c7060bb3/tests/test_core.py#L9460-L9461

However, this will not be automatically run by CI, the test target needs to be added manually in the config.

Reply:
I'd rather not move to core, as that is generally meant for tests we run across all opt levels etc., which isn't necessary here.

I think a separate function might be the best thing for now here, if the decorators do not work atm.